commands/bootstrap/__tests__/__snapshots__/bootstrap-command.test.js.snap

```diff
@@ -319,9 +319,6 @@ Object {
     "bar@^2.0.0",
     "foo@^1.0.0",
   ],
-  "packages/package-4": Array [
-    "package-3@^1.0.0",
-  ],
```

Comment:
Here!

This snapshot shows which dependencies were resolved from the npm registry. 

Before my change, the dependency of `packages/package-4` on `package-3` was resolved from the npm registry.

After my change, `package-3` is resolved as a link within monorepo. (See changes in the second snapshot below.)